contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
+	  font-family: Verdana, sans-serif;
+	}
+	
+	.center{
```

Comment:
Any style that is written as ".[enter name here]" is called a class. You can add it to tags. For example, I added this to the <header> tag (see below)